clicommand/tool_sign.go

```diff
@@ -60,65 +61,63 @@ var ToolSignCommand = cli.Command{
 	Usage: "Sign pipeline steps",
 	Description: `Usage:
 
-    buildkite-agent tool sign-pipeline [options...]
+    buildkite-agent tool sign-pipeline [options...] [pipeline-file]
 
 Description:
 
 This (experimental!) command takes a pipeline in YAML format as input, and annotates the
 appropriate parts of the pipeline with signatures. This can then be input into the YAML steps
-editor in the Buildkite UI so that the agents running these steps can verify the signatures.`,
+editor in the Buildkite UI so that the agents running these steps can verify the signatures.
+
+If a token is provided using the ′graphql-token′ flag, the tool will attempt to retrieve the
+pipeline definition and repo using the Buildkite GraphQL API. If ′update′ is also set, it will
+update the pipeline definition with the signed version using the GraphQL API too.`,
 	Flags: []cli.Flag{
 		cli.StringFlag{
-			Name:     "graphql-token",
-			Usage:    "A token for the buildkite graphql API. This will be used to populate the pipeline invariants if they are not provided.",
-			EnvVar:   "BUILDKITE_GRAPHQL_TOKEN",
-			Required: false,
+			Name:   "graphql-token",
+			Usage:  "A token for the buildkite graphql API. This will be used to populate the value of the repository URL, and download the pipeline definition. Both ′repo′ and ′pipeline-file′ will be ignored in preferance of values from the GraphQL API if the token in provided.",
```

Comment:
tyop:

```suggestion
			Usage:  "A token for the buildkite graphql API. This will be used to populate the value of the repository URL, and download the pipeline definition. Both ′repo′ and ′pipeline-file′ will be ignored in preference of values from the GraphQL API if the token in provided.",
```